helpers/webhook.cfc

```diff
@@ -0,0 +1,252 @@
+component accessors="true" {
+
+    property name="url" default="";
+    property name="bounce" default=true type="boolean";
+    property name="click" default=true type="boolean";
+    property name="deferred" default=true type="boolean";
+    property name="delivered" default=true type="boolean";
+    property name="dropped" default=true type="boolean";
+    property name="enabled" default=true type="boolean";
+    property name="group_resubscribe" default=false type="boolean";
+    property name="group_unsubscribe" default=false type="boolean";
+    property name="open" default=true type="boolean";
+    property name="processed" default=true type="boolean";
+    property name="spam_report" default=true type="boolean";
+    property name="unsubscribe" default=false type="boolean";
+    property name="oauth_client_id" default="";
+    property name="oauth_client_secret" default="";
+    property name="oauth_token_url" default="";
+
+  
+    /**
+    * @hint Allow all values to be set by the init.   The URL is the only required key needed to create a webhook.
```

Comment:
The comment says that all values can be set in the init, but it looks like it's only the url. Can we just update the comment to say that it's only the URL that can be set in the init?

Reply:
I have updated the comment and pushed the changed